FWCore/Framework/src/ScheduleItems.cc

```diff
@@ -144,13 +151,4 @@ namespace edm {
                                       config,
                                       processContext);
   }
-
-  void ScheduleItems::clear() {
-    // propagate_const<T> has no reset() function
-    actReg_ = nullptr;
-    preg_ = nullptr;
-    branchIDListHelper_ = nullptr;
-    thinnedAssociationsHelper_ = nullptr;
-    processConfiguration_ = nullptr;
-  }
```

Comment:
Is this function not being called?

Reply:
Yes. I noticed it was not called anywhere in CMSSW.

Reply:
Move this deletion to a separate PR. #34406